examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c

```diff
@@ -527,10 +528,11 @@ static inline int8_t system_init( void )
     /* configure the timers  */
     Timer_Config();
 
-    UART_Configuration( WIFI_BOARD_UART_BAUDRATE );
+    UART_Configuration( WIFI_BOARD_UART_BAUDRATE ); /* From stm32_spwf_wifi.c */
 #ifdef USART_PRINT_MSG
     UART_Msg_Gpio_Init();
-    USART_PRINT_MSG_Configuration( DEBUG_UART_BAUDRATE );
+    if ( 0 > USART_PRINT_MSG_Configuration( DEBUG_UART_BAUDRATE ) )

```

Comment:
Good point. I believe the SDK's style is inconsistent. I've definitely seen some single-line statements without brackets, but I'm happy to conform to the libxively style. It will make some functions unnecessarily lengthy, but oh well.

What would you rather see?